index.d.ts

```diff
@@ -0,0 +1,50 @@
+export as namespace EventEmitter;
```

Comment:
I don't think that shipping typings that use ES6-style imports will break anyone's existing code. 

[From TS design discussion](https://github.com/Microsoft/TypeScript/issues/2338):

``` js
If exists ambient external module named X {
  return the ambient external module 
}
else if X begins with './' or '../' or it is rooted path {
  try LOAD_AS_FILE(Y + X, loadOnlyDts=false)
  try LOAD_AS_DIRECTORY(Y + X, loadOnlyDts=false)
}
else {
  LOAD_NODE_MODULES(X, dirname(Y))
}
THROW "not found"
```

So if a user already has `.d.ts` files installed for eventemitter3, the definitions in this node module would not be used. This is consistent with what I tested locally.